src/wash/wash.hpp

```diff
@@ -198,4 +201,35 @@ namespace wash {
         Compute the euclidean distance between particles
     */
     double eucdist(const Particle& p, const Particle& q);
+
+    /**
+     * @brief Helper function to format to a c++ string
+     * 
+     * @tparam Args 
+     * @param format standard style format string
+     * @param args arguments for the format string
+     * @return std::string 
+     */
+    template <typename... Args>
+    std::string string_format(const std::string& format, Args... args) {
```

Comment:
Could we keep these helper functions in a separate file like the previous `wash_extra_solutions.cpp`?

Reply:
Sure. Moved to a separate file.